README.md

```diff
@@ -74,8 +74,8 @@ require("zk").setup()
 **The default configuration**
 ```lua
 require("zk").setup({
-  -- can be "telescope", "fzf", "fzf_lua" or "select" (`vim.ui.select`)
-  -- it's recommended to use "telescope", "fzf" or "fzf_lua"
+  -- can be "telescope", "fzf", "fzf_lua", "minipick", or "select" (`vim.ui.select`)
```

Comment:
what about naming it `mini.pick` or `mini` instead?

Reply:
Sure, I'll rename it to `mini.pick`.

Reply:
I recall why I named it `minipick`. Because that option is just concatenated to `"zk.pickers"` and then required. So, if we rename that, then we need to rename the file as well, but I don't think you can have dots in the filename, right? And I prefer to name it "minipick" over "mini" as many users only use a subset of the mini umbrella.